### DISCUSSION TITLE
Option to set default target branch of pull requests in a repository

### DISCUSSION THREAD

**Discussion by YanisBft** [community/community#17681] [Pull Requests]
When you open a PR, the targeted branch by default is the main branch of the repository. If you don't want to push on the main branch, you always have to select the branch you want.
In case you never want to target the main branch, but rather a "dev" branch for instance, it would be helpful to have the option to set the default target branch of pull requests.

**Comment by dpkirchner** [comment]
This is the same request as was created in the old feedback "forum": https://github.com/isaacs/github/issues/1401. I'm sure it's not possible to bring over all the thumbs-ups from there, but if it were, this request would easily be in the top 10 requests ever made on the discussions forum.
